examples/tut-10/App.js

```diff
@@ -10,61 +10,59 @@ import React from 'react';
 import AppViews from './src/AppViews';
 import DeployerViews from './src/DeployerViews';
 import AttacherViews from './src/AttacherViews';
-import { renderView } from './src/render';
+import {renderView} from './src/render';
 import * as backend from './build/index.main.mjs';
-import { loadStdlib } from '@reach-sh/stdlib';
-import { Platform } from 'react-native';
+import {loadStdlib} from '@reach-sh/stdlib';
 
 const reach = loadStdlib(process.env);
 
-const handToInt = { ROCK: 0, PAPER: 1, SCISSORS: 2 };
+const handToInt = {ROCK: 0, PAPER: 1, SCISSORS: 2};
 const intToOutcome = ['Bob wins!', 'Draw!', 'Alice wins!'];
-const { standardUnit } = reach;
-const defaults = { defaultFundAmt: '10', defaultWager: '0.005', standardUnit };
+const {standardUnit} = reach;
+const defaults = {defaultFundAmt: '10', defaultWager: '0.005', standardUnit};
 
 class App extends React.Component {
   constructor(props) {
     super(props);
-    this.state = { view: 'ConnectAccount', ...defaults };
+    this.state = {view: 'ConnectAccount', ...defaults};
     const init = async () => {
       await reach.setProviderByName('TestNet');
-    }
+    };
```

Comment:
@mobileguru007 are you using the Eslint extension for Visual Studio Code? It'll catch missing semicolons for you automatically if your project is using `eslint` and you have the semicolon rule enabled.

Reply:
I just realized 3f5c4a6 has the commit message `Eslinting Done`, so we are using `eslint` here.

I think that'll be helpful to us long-term.